web/src/routes/(user)/search/[[photos=photos]]/[[assetId=id]]/+page.svelte

```diff
@@ -144,6 +144,7 @@
       page: nextPage,
       withExif: true,
       isVisible: true,
+      language: $lang,
```

Comment:
Should we add a new setting? I like to have my UI in English, but I prefer to use my native language for searching.

Reply:
Sure, we could have a Search Language user preference separate from the UI language.

Reply:
Would it make sense to also have this be something the admin can override?

Reply:
Do you mean making it use a certain search language by default for all users? That could be nice, possibly doing the same for the UI language as well. But I think we should avoid making too many changes at once in this PR, so the admin stuff can be done separately.

Reply:
A few related FRs came up that I consolidated into #13651.